packages/plans-grid-next/src/components/shared/header-price/index.tsx

```diff
@@ -298,4 +293,4 @@ const PlanFeatures2023GridHeaderPrice = ( {
 	);
 };
 
-export default PlanFeatures2023GridHeaderPrice;
+export default HeaderPrice;
```

Comment:
Should the classes added in this file be renamed and the styles migrated along in a locally imported stylesheet? (I think so)

For class naming, you'd be prefixing with `plans-grid-next` - see other cases cleaned up recently (e.g. storage add-ons)

Reply:
Yes good catch, I'll migrate the styles and incorporate the rename in a followup.

Reply:
Followup: https://github.com/Automattic/wp-calypso/pull/93357